src/databases/CGNS/CGNSCommonPluginInfo.C

```diff
@@ -64,6 +67,75 @@ CGNSCommonPluginInfo::SetupDatabase(const char *const *list,
 {
     avtDatabase *db = NULL;
 
+    //
+    // If the number of blocks is 1, see if the names follow the
+    // conventions for a multi-file multi-block file.
+    //
+    if (nBlock == 1)
+    {
+        bool multiblockFile = true;
+        int  globalNBlocks = -1;
+        for (int f = 0 ; multiblockFile && f < nList; f++)
+        {
+            //
+            // Eliminate the path information to get just the filename.
+            //
+#if defined _WIN32      
+            const char *file = strrchr(list[f], '\\');
+#else
+            const char *file = strrchr(list[f], '/');
```

Comment:
Is this dealing with a filesystem path or a path internal to a cgns file?
If filesystem, the test can be simplified to a single line utilizing VISIT_SLASH_CHAR.
If internal, both forms of path separator may need to be tested.  Can I have access to data to test on Windows?